packages/sdk/src/client.ts

```diff
@@ -584,10 +584,10 @@ export function SoundClient({
   }) {
     const minterAddresses = await editionRegisteredMinters({
       editionAddress,
-      fromBlockOrBlockHash,
+      fromBlockOrBlockHash: 0, // This should be zero because we want to get info from all the registered minters
```

Comment:
no? if the user is giving a explicit block it should 100% be respected

Reply:
at most you could put a default value in the parameter

Reply:
If we don't put zero here, we fail to include mintIds that were created _after_ the specified block by minters that were deployed _before_ the block.

Reply:
Actually, the number we should have here is the earliest minter could have been deployed. I can come back and update in another PR: https://linear.app/sound/issue/ENG-2059/add-earliest-possible-block-numbers-to-sdk

Reply:
no mat, it's still not clear, if a user gives a block that block is used

Reply:
if the user puts a bad block it's the user mistake, not the responsibility of the sdk

Reply:
If I'm asking the SDK "get me all mintIds created after `x` block", I don't care when the minters were created. All I care about is I get all the mintIds after that block. Forcing the user to understand our implementation details seems like it will create a lot of confusion.

Reply:
> but the providers don't always allow looking up from the block 0

Addressed that here:
https://github.com/soundxyz/sdk/pull/136/commits/bf31c7588203b7e5bf7576dbd93e45393d65427b

> you can't assume you can simply override the user's block parameter

I'm not overriding it for the mintIds, which is the only thing relevant to the user calling this function.

Reply:
@PabloSzx Sorry - I was misunderstanding you. Spoke to Vignesh and cleared it up. 

Given that `fromBlockOrBlockHash` is optional, it makes sense that we respect it in all cases. Made a note in the docs in case it creates any confusion: https://github.com/soundxyz/docs/pull/50